lib/sandbox/pmapi.js

```diff
@@ -110,7 +110,34 @@ Postman = function Postman (bridge, execution, onRequest, cookieStore) {
         /**
          * @type {CookieList}
          */
-        cookies: execution.cookies
+        cookies: execution.cookies,
+
+        /**
+         * @type {Object}
+         */
+        visualiser: {
+            /**
+             * Set visualiser template and its options
+             *
+             * @param {String} template - visualisation layout in form of template
+             * @param {Object} [data] - data object to be used in template
+             * @param {Object} [options] - options to use while processing the template
+             */
+            set: function (template, data, options) {
```

Comment:
Need to enforce data and options are objects

Reply:
How about we do this check in runtime while processing template?
If not then what should be done? ignore it or throw error?

Reply:
Throw error